generate_labels.py

```diff
@@ -11,6 +11,8 @@
 ydn_reverse_map = {0: 'y', 1: 'd', 2: 'n'}
 
 for file_name in all_file_names:
+    if 'boundary_box' in file_name:
```

Comment:
We need to think of a way to manage this, the bounding boxes made by different people might differ.